### ISSUE TITLE
:line-length warnings can't be :clj-kondo/ignore-ed

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#2236]
``` clojure
(ns whitespace)

#_{:clj-kondo/ignore [:line-length]}
#_:clj-kondo/ignore
(def example "this line of code exceeds 80 characters but shouldn't cause warning")
```
